MainProject/Company.cs

```diff
@@ -59,19 +81,24 @@ public void PrintAllShops()
             }
         }
 
-        public void PrintCompanyEmployeesCountByCity(string city)
+        public virtual void PrintCompanyEmployeesCountByCity(string city)
         {
             int companyEmployeeCount = 0;
             foreach (Shop shop in shops)
             {
-                if (shop.city == city)
+                if (shop.City == city)
                 {
-                    companyEmployeeCount += shop.employeeCount;
+                    companyEmployeeCount += shop.EmployeeCount;
                 }
             }
 
             Console.WriteLine($"Count of shops employees: {companyEmployeeCount}");
```

Comment:
Please add also criteria (city) in output in console

Reply:
And company as well

Reply:
Done